packages/vision/src/helpers.js

```diff
@@ -73,14 +73,27 @@ var coerceImage = (image, callback) => {
  * @return {Function} - The function that, when called, will call annotateImage
  *   asking for the single feature annotation.
  */
-var _createSingleFeatureMethod = featureValue => {
-  return function(image, options) {
-    return this.annotateImage({
-      image: image,
-      features: [{type: featureValue}],
-    }, options);
-  };
-};
+ var _createSingleFeatureMethod = featureValue => {
+   return function(annotateImageRequest, callOptions) {
+     annotateImageRequest.features = annotateImageRequest.features || [];
+
+     // Ensure the feature value indicated by the user's method choice exists on the
+     // features array; if it does not, add it.
+     var found = false;
```

Comment:
I think this is solved with documentation, and I still see your example as a misuse of the method. The method should be documented as "the way to get crop hints" and not the way to get arbitrary features, because we already have another method for exactly that. `cropHints()` would not document the features property, because that's the part this method controls. If we have such a loose definition of what this method is for, I would lean back towards scrapping the shorthand methods. "I called cropHints() and I got labels! You had one job!"

> as a user, I wonder why I did not get something that I asked for.

By not documenting the `features` property, the user would be misusing the method, which would address why they're not getting something *they think* they're asking for. (What they're actually asking for is "cropHints()" and they will get that)

> someone decides they also want faces, and in looking up how to do it in docs, stumbles on the features array.

But they would have stumbled upon it on another method's documentation.